.circleci/config.yml

```diff
@@ -39,71 +127,61 @@ jobs:
             cd git-secrets-1.3.0
             sudo make install
       - run:
-          name: run the actual tests
-          command: mvn -B clean install -DskipTests
-      - run:
-          name: validate openapi
+          name: decrypt
           command: |
-            wget --no-verbose https://repo.maven.apache.org/maven2/org/openapitools/openapi-generator-cli/4.3.0/openapi-generator-cli-4.3.0.jar -O openapi-generator-cli.jar
-            java -jar openapi-generator-cli.jar validate -i dockstore-webservice/src/main/resources/swagger.yaml
-            java -jar openapi-generator-cli.jar validate -i dockstore-webservice/src/main/resources/openapi3/openapi.yaml
+            sudo apt install openssl -y
+            openssl aes-256-cbc -d -in circle_ci_test_data.zip.enc -k $CIRCLE_CI_KEY -iv $CIRCLE_CI_IV >> test_data.zip
+            bash expand_confidential_bundle.sh test_data.zip abandonDatabaseConnectionsWithOrcid
+      - run:
+          name: build
+          command: mvn -B clean install -DskipTests
+      # - run:
+      #     name: validate openapi
+      #     command: |
+      #       wget --no-verbose https://repo.maven.apache.org/maven2/org/openapitools/openapi-generator-cli/4.3.0/openapi-generator-cli-4.3.0.jar -O openapi-generator-cli.jar
+      #       java -jar openapi-generator-cli.jar validate -i dockstore-webservice/src/main/resources/swagger.yaml
+      #       java -jar openapi-generator-cli.jar validate -i dockstore-webservice/src/main/resources/openapi3/openapi.yaml
   unit_test: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
-      - image: circleci/openjdk:11.0.4-jdk-stretch
+      - image: cimg/openjdk:11.0.8
         environment:
           JAVA_TOOL_OPTIONS: -Xmx512m # Java can read cgroup. Sadly the cgroup in CircleCI is wrong. Have to manually set. Nothing to do with surefire plugin, it has its own JVM. The two of these must add up to a bit less than 4GB.
           PGHOST: 127.0.0.1
-      - image: circleci/postgres:9.6.5-alpine-ram
+      - image: circleci/postgres:9.6.19
```

Comment:
Postgress 11? Or is there a reason you left it on 9?

Reply:
`Unit tests still uses postgres 9.x.x, it requires migration changes (splitting to different ticket)`